internal/services/dynatrace/dynatrace_monitor_resource.go

```diff
@@ -167,6 +168,27 @@ func (r MonitorsResource) Arguments() map[string]*pluginsdk.Schema {
 			},
 		},
 
+		"environment_properties": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"environment_info": {
+						Type:     pluginsdk.TypeList,
+						Required: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"environment_id": {
```

Comment:
Why are we nesting 3 levels here for a single property/value?

Reply:
There are other properties will be supported recently under them.